nntrainer/tensor/manager.cpp

```diff
@@ -120,13 +120,16 @@ MMapedMemory::~MMapedMemory() {
   ml_logd("[MMapedMemory] buf released");
 }
 
-Manager::Manager(bool enable_gradient_memory_opt_, bool use_shared_memory_) :
+Manager::Manager(bool enable_gradient_memory_opt_,
+                 bool enable_derivative_memory_opt_,
```

Comment:
Please check https://github.com/nnstreamer/nntrainer/pull/830#discussion_r549601909

Reply:
Yes this is affecting the unittest, I'll make another commit to resolve the issue :)